added v1 README.md to the experiment folder (#1)
s-ccs/2024FreeViewingMSCOCO: experiment/README.md

|                                    Break start                                    |          5         |                                 send_trigger-breakstart                                 |
|                                     Break end                                     |          6         |                                  send_trigger-breakend                                  |
|                                  End of practice                                  |          7         |                                End_of_practice sketchpad                                |
|                                 Stimulus event end                                |          8         |                                send_trigger_end_stimulus                                |
|                                 Manual pause start                                |          9         |                             send_trigger_manual_pause_start                             |

### Runif conditions used

> Run if refers to the field in opensesame in the `sequence` item of the trial loop. It is used to send triggers only when the condition is met.

Reviewer: ```suggestion
`Run if` refers to the field in OpenSesame in the `sequence` item of the trial loop. It is used to execute certain items (e.g. breaks or sending triggers) only when the condition is met.
```

#### Main block sequence

> | Opensesame Object |                                                        Runif                                                       |   |

Reviewer: ```suggestion
| OpenSesame Object |                                                        Runif                                                       |   |
```

|:-----------------:|:------------------------------------------------------------------------------------------------------------------:|---|
|   break_sequence  | =(count_block_sequence&gt;0) and (count_block_sequence!=total_trials) and ((count_block_sequence%block_size) == 0) |   |

> #### fixation block sequence

Reviewer: ```suggestion
#### Fixation block sequence
```

> |      **Opensesame Object**      |                                  **Runif**                                 |

Reviewer: ```suggestion
|      **OpenSesame Object**      |                                  **Runif**                                 |
```

# FreeViewingMSCOCO 

> This is a free-viewing EEG-ET experiment written using Opensesame.

Reviewer: ```suggestion
This is a free-viewing EEG-ET experiment written using [OpenSesame](https://osdoc.cogsci.nl/).
```

## Installation 👩‍💻

1. Install the required packages via conda. You can create a new environment using the `environment.yaml` file as follows:
```bash
conda env create -f environment.yaml
```

> [!NOTE]  

> > Here we install opensesame via conda. If you want to install via other download options, please refer to the [official website](https://osdoc.cogsci.nl/3.2/download#all-download-options).

Reviewer: ```suggestion
> Here we install OpenSesame via conda. If you want to install it via other download options, please refer to the [official website](https://osdoc.cogsci.nl/3.2/download#all-download-options).
```

2. EyeLink 1000 Plus Setup
    - Connect the EyeLink 1000 Plus to the host PC and follow the instructions in the manual to set up the device. [More on this later].
    - Make sure that the EyeLink 1000 Plus is connected to the host PC via the Ethernet cable/wifi.

## Experiment Flow 🌊

[WILL BE UPDATED WITH A FLOWCHART]

The structure of the experiment is as follows:

> 1. Welcome and Preliminary Instructions

Reviewer: Why "preliminary"? These are the actual instructions, aren't they? 
I would remove the "preliminary" in the document as well as in the naming in the OS experiment.

Author: Yes, these are actual ones. Will make the changes

2. Initial Calibration
3. Start trial loop

    a. Press SPACE to start the trial

    b. Fixation cross

>     c. Wait for center gaze fixation (if not recalibrate and repeat the trial)

Reviewer: ```suggestion
    c. Wait for center gaze fixation (if not, recalibrate and repeat the trial)
```

>     d. Jitter (fixation still shows up on the screen)

Reviewer: ```suggestion
    d. Jitter (fixation cross still shows up on the screen)
```

|  Stimulus Image Shown |          2         |                             send_trigger_start_stimulus                            |                                 |
|   Calibration Start   |          3         |                wait_for_centre_gaze (Prepare) - at calibration step                |                                 |
|    Calibration End    |          4         |               wait_for_centre_gaze (Prepare) - after calibration step              |                                 |
| Mandatory_Break Start |          5         |                               send_trigger-breakstart                              | =(count_block_sequence+1)%50==0 |
|  Mandatory_Break End  |          6         |                                send_trigger-breakend                               | =(count_block_sequence+1)%50==0 |
|    End of Practice    |          7         |                              End_of_practice sketchpad                             |                                 |
|   Stimulus event end  |          8         |                              send_trigger_end_stimulus                             |                                 |

> |   Manual Pause Start  |          9         |                          `send_trigger_manual_pause_start`                         |  c for calibrate; r for resume  |

Reviewer: I like that you want to add the information about which keys are allowed in manual pauses (i.e.  c for calibrate; r for resume). However, it does not really fit in "Run if", since it's run if the key "p" is pressed, or am I overlooking anything?

Author: Yes, this table with Runif is a bit confusing. Rather, we can have a Runif table separately corresponding to the eye tracker objects used.

> |   Manual Pause Start  |          9         |                          `send_trigger_manual_pause_start`                         |  c for calibrate; r for resume  |

Reviewer: I like having the trigger table. Could you make sure that the `trigger names` and the `OpenSesame location` in the table match the ones from the OpenSesame experiment? And is it on purpose that some trigger names contain "_" while others don't? Because this might be important when parsing the triggers later on.

Author: Yes, we just synchronized both the experiment and trigger the table. Also made the structure consistent.
Thank you for the suggestion

|    Manual Pause End   |         10         |                           `send_trigger_manual_pasue_end`                          |                                 |

**Run if refers to the field in opensesame in the `sequence` item of the trial loop. It is used to send triggers only when the condition is met.*

> > [!CAUTION] Never name any variable in your inline script as `timeout` 🥲. It might break the functionality of your experiment. Follow this [discussion](https://forum.cogsci.nl/discussion/6393/sketchpad-does-not-wait-for-the-keypress) for more details!

Reviewer: I'm not sure whether this was meant to be a callout box (as with the note) but it does not seem to show up in the rendered version. Maybe it's not "caution" but "warning" but I'm not sure.

Author: CAUTION usually works, I will recheck and make it WARNING.
